boost-gradle/src/main/groovy/io/openliberty/boost/gradle/tasks/BoostPackageTask.groovy

```diff
@@ -167,6 +191,48 @@ public class BoostPackageTask extends AbstractBoostTask {
         return classifier
     }
 
+    protected List<BoosterPackConfigurator> getBoosterConfigsFromDependencies() {
+        List<String> dependencyList = new ArrayList<String>()
+
+        def componentIds = project.configurations.compile.incoming.resolutionResult.allDependencies.collect { it.selected.id }
+        def result = project.dependencies.createArtifactResolutionQuery()
+            .forComponents(componentIds)
+            .withArtifacts(MavenModule, MavenPomArtifact)
+            .execute()
+
+        for (component in result.resolvedComponents) {
+            logger.info("Adding ${component.id} to dependency list.")
+            component.getArtifacts(MavenPomArtifact).each { dependencyList.add(component.id.toString()) }
+        }
+
+        return boosterParent.mapDependenciesToFeatureList(dependencyList)
```

Comment:
Maybe slightly better to change signature to return dependencyList, and pass that onto `generateServerXMLJ2EE` ?    The Maven side is doing the same thing...... the core Maven vs. Gradle diff is just building the dependency list.    

Maybe we want to defer building a feature list just yet and wait until we start to generate.